package.json

```diff
@@ -23,43 +26,30 @@
 		"postinstall": "npm run build-prod",
 		"start": "npm run build-dev-once && concurrently --kill-others \"npm run api-dev\" \"npm run build-dev\"",
 		"storybook": "start-storybook -p 9001 -c .storybook -s ./",
-		"test": "npm run lint && npm run check && jest --forceExit --silent",
+		"test": "npm run lint && jest --forceExit --silent",
 		"test-dev": "jest --forceExit",
-		"tools": "NODE_PATH=./client:./ node -r esm --max-old-space-size=12000 ./tools",
-		"workers-dev": "SEQUELIZE_MAX_CONNECTIONS=1 NODE_PATH=./client:./ WORKER=true nodemon workers/init.js --watch workers",
-		"workers-prod": "SEQUELIZE_MAX_CONNECTIONS=1 NODE_PATH=./client:./ WORKER=true node workers/init.js"
+		"tools": "NODE_PATH=./client:./ node -r ts-node/register -r esm --max-old-space-size=12000 ./tools",
```

Comment:
**Important change** — Tools are now run with `ts-node` since many tools import TS code from the server (namely the sequelize models).